src/styles/saas-list.less

```diff
@@ -56,21 +96,45 @@
           font-size: 32px;
         }
       }
+
       .card-title {
-        vertical-align: middle;
-        display: table-cell;
+        color: inherit;
         padding-left: 8px;
       }
     }
-    .card-footer {
-      display: table-cell;
-      vertical-align: middle;
-      height: 80px;
 
-      .card-description {
-        font-size: 14px;
+    @media (min-width: @screen-sm-min) { // for card-container
+      display: block;
+
+      .card-header {
+        display: block;
+        text-align: center;
+        margin-bottom: 10px;
+
+        .card-icon {
+          font-size: 108px;
+          width: 100%;
+          margin-bottom: 20px;
+
+          &.fa.fa-mobile {
```

Comment:
My recommendation is to use SVG images as I believe the icons in the mocks are part of the [standard brand icons](http://brand.redhat.com/elements/icons/), which are in SVG format, but not available as a font (which is really no longer needed for our purposes since our supported browsers support SVG).